content/blog/2024-09-12-meet-the-board-jack.md

```diff
@@ -9,7 +9,7 @@ date: '2024-09-12'
 images:
   - /blog-images/2024/jack_qanda.png
 post:
-    title: ""
+    title: "Meet the board - Jack Aboutboul"
```

Comment:
Wrong name

Reply:
that's on the Jack blog post, so I don't think it's wrong. Am I missing something?